test/test_utils.py

```diff
@@ -90,9 +90,10 @@ def test_draw_boxes(self):
 
         path = os.path.join(os.path.dirname(os.path.abspath(__file__)), "assets", "fakedata", "draw_boxes_util.png")
         if not os.path.exists(path):
-            write_png(result, path)
+            res = Image.fromarray(result.permute(1, 2, 0).contiguous().numpy())
+            res.save(path)
```

Comment:
This was recently changed based on [your comment](https://github.com/pytorch/vision/pull/2785#discussion_r531621316). What's the issue using the io.image here?

Reply:
Right, my bad sorry about that.

Internal fb tests were failing because this test now imposes that the image part of torchvision is compiled, and we would need to adapt some TARGETS file for that. In order to move forward with fbcode integration, it's easier to first change to using PIL, then update the TARGETS file in fbcode and then add back the dependency to `torchvision.io`.